Neos.Neos/Classes/Command/UserCommandController.php

```diff
@@ -88,7 +88,7 @@ public function showCommand(string $username, string $authenticationProvider = n
 
         $this->outputLine('<b>First name:</b> %s', [$user->getName()->getFirstName()]);
         $this->outputLine('<b>Last name:</b> %s', [$user->getName()->getLastName()]);
-        $this->outputLine('<b>Backend language:</b> %s', [$user->getPreferences()->getInterfaceLanguage() ?? 'Use system default']);
```

Comment:
I have noticed a bug:

`??` is the wrong operator here because it accepts null as a valid value and therefore it does not output the fallback if the system backend language is configured for the user in the backend.

Instead, `?:` is correct because it uses the fallback even if the value is null or empty.